scylla/src/frame/response/result.rs

```diff
@@ -96,7 +95,7 @@ pub enum CqlValue {
     UserDefinedType {
         keyspace: String,
         type_name: String,
-        fields: BTreeMap<String, Option<CqlValue>>,
+        fields: Vec<(String, Option<CqlValue>)>,
```

Comment:
Here, please add a remark that the order of fields in the vector must match the order of fields as defined in the UDT and that the driver does not check this by itself, so incorrect data will be written if the order is wrong.

Reply:
Done